week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,17 +1,18 @@
-// Add comments to explain what this function does. You're meant to use Google!
+// returns a random number between 0 (inclusive),  and 1 (exclusive)
 function getNumber() {
     return Math.random() * 10;
 }
 
-// Add comments to explain what this function does. You're meant to use Google!
+// to join two or more arrays.
```

Comment:
That's correct. Which other type of value other than arrays would this function allow us to concatenate?